From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Mon, 31 Dec 2018 16:17:55 -0500
Subject: universal-access: Reverse Repeat Keys Speed direction

Since the gsettings is for the repeat-interval but the
GUI is for Speed, we need to reverse the mapped direction.

We have to turn off the fill indicator (has_origin) for Speed
since it fills the right side instead of the left. We turn
off the indicator for Delay to match.

Closes: https://gitlab.gnome.org/GNOME/gnome-control-center/issues/311
(cherry picked from commit c61a9977f10d78a62080d28d6db768347fece18b)
---
 panels/universal-access/uap.ui | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/panels/universal-access/uap.ui b/panels/universal-access/uap.ui
index 3c0622a..f417e9e 100644
--- a/panels/universal-access/uap.ui
+++ b/panels/universal-access/uap.ui
@@ -1538,6 +1538,7 @@
                 <property name="hexpand">True</property>
                 <property name="adjustment">repeat_delay_adjustment</property>
                 <property name="draw_value">False</property>
+                <property name="has_origin">False</property>
                 <property name="width-request">400</property>
                 <child internal-child="accessible">
                   <object class="AtkObject" id="repeat_delay_scale-atkobject">
@@ -1623,6 +1624,8 @@
                 <property name="hexpand">True</property>
                 <property name="adjustment">repeat_speed_adjustment</property>
                 <property name="draw_value">False</property>
+                <property name="has_origin">False</property>
+                <property name="inverted">True</property>
                 <property name="width-request">200</property>
                 <child internal-child="accessible">
                   <object class="AtkObject" id="repeat_speed_scale-atkobject">
